8dev_restapi.js

```diff
@@ -235,6 +319,12 @@ class Service extends EventEmitter {
     });
   }
 
+  /**
+   * Stops authentication timer,
+   * shuts down socket listener and deletes notificatrion callback,
```

Comment:
Reikėtų aprašyti ką daro funkcija, o ne kokius konkrečiai veiksmus atlieka. Kažkas iš serijos `Stops receiving and processing events`/`Stops this service and and all it's subservices that were started in start(). Cleans up resources`